docs/getting-started-guides/rkt/index.md

```diff
@@ -93,8 +176,8 @@ If you are using the [hack/local-up-cluster.sh](https://github.com/kubernetes/ku
 
 ```shell
 $ export CONTAINER_RUNTIME=rkt
-$ export RKT_PATH=$PATH_TO_RKT_BINARY
-$ export RKT_STAGE1_IMAGE=$NAME_OF_THE_STAGE1_IMAGE
+$ export RKT_PATH=<rkt_binary_path>
```

Comment:
Comment mentioning that both of these are optional if your rkt is in your path and well configured would be nice.

e.g.

`# RKT_PATH and RKT_STAGE1_IMAGE are optional if rkt is in your path with a default stage1`